src/OrbitDB.js

```diff
@@ -239,7 +265,7 @@ let databaseTypes = {
     delete this.stores[address]
   }
 
-  async _determineAddress(name, type, options = {}) {
+  async _determineAddress(name, type, options = {}, onlyHash) {
```

Comment:
Why remove `onlyHash` from the options? It needs to be passed to `createDBManifest`. I believe its an artifact that we have a `_determineAddress` and `determineAddress` I think we can get rid of one (I could be missing sth)